bokeh/models/widgets/widget.py

```diff
@@ -7,8 +7,11 @@
 from __future__ import absolute_import
 
 from ...core.properties import abstract
+from ...core.properties import String, List
 from ..component import Component
 
 @abstract
 class Widget(Component):
     """ A base class for all interactive widget types. """
+
+    css_classes = List(String)
```

Comment:
Also, not really a `String`. There should be `CSSClass` property descriptor implemented on top of `Regex`.